R/prettyRainfallPlot.R

```diff
@@ -212,7 +211,8 @@ prettyRainfallPlot = function(this_sample_id,
 
   if (label_sv) {
     message("Getting combined manta + GRIDSS SVs using GAMBLR ...")
-    these_sv = get_manta_sv(these_sample_ids  = this_sample_id)
+    these_sv = get_manta_sv(these_sample_ids = this_sample_id, 
```

Comment:
Are you planning to add the non-annotated sv support here or it will be separate?

Reply:
I think this PR is good to go, I can add this new functionality in an upcoming PR. That way, this PR will be more related to resolving bugs whereas the next would introduce function improvements.